util/jsonutil/jsonutil.go

```diff
@@ -211,3 +213,44 @@ func tryExtractErrorMessage(err error) string {
 func isLikelyDetailedErrorMessage(msg string) bool {
 	return !strings.HasPrefix(msg, "request.")
 }
+
+// RawMessageExtension will call json.Compact() on every json.RawMessage field when getting marshalled.
+// All other types will be marshalled as usual
```

Comment:
Recommend removing the "All other types will be marshalled as usual" part. Doesn't seem necessary.

Reply:
Removed